samples/Config.ChangeNotification.Web/project.json

```diff
@@ -1,15 +1,12 @@
 {
   "dependencies": {
     "Microsoft.AspNetCore.Http": "1.0.0-*",
-    "Microsoft.Extensions.Configuration": "1.0.0-*",
-    "Microsoft.NETCore.Platforms": "1.0.1-*"
+    "Microsoft.Extensions.Configuration": "1.0.0-*"
```

Comment:
Might be because Microsoft.AspNetCore.Http or Microsoft.Extensions.Configuration brought it in